index.html

```diff
@@ -32,7 +32,24 @@
         <li>Contact</li>
       </ul>
     </nav>    
-  </header>  
+  </header> 
+
+  <div id="myNav_mobile" class="overlay-bg">
+
+    <!-- Button to close the overlay navigation -->
+    <a class="closebtn">
+      <img src="./images/mobile_menu/Icon.png" alt="Close_icon">
+    </a>
+  
+    <!-- Overlay content -->
+    <ul class="overlay-content">
+      <li class="overlay-mobile"><a href="#home">Portfolio</a></li>
```

Comment:
- [ ] You have done a good job with your mobile menu :100: but I would suggest updating this "Portfolio" link because one of the requirements it's that a link should display the correct part of the page and this link should show the "Project" sections. Please, kindly update this link as it is required :smiley:
![image](https://user-images.githubusercontent.com/81522435/145887311-692fa5a9-b75a-40c2-bc85-d39f33895633.png)

Reply:
:+1: Addressed :heavy_check_mark: